src/cam_touptek.cpp

```diff
@@ -297,6 +337,10 @@ bool CameraToupTek::EnumCameras(wxArrayString& names, wxArrayString& ids)
         names.Add(ti[i].displayname);
         ids.Add(ti[i].id);
     }
+    if (numCameras > 0)
+    {
+        SelectCamera(ids[0]);
+    }

```

Comment:
I don't think it is a good idea to automatically select camera 0 here.  EnumCameras is called when the user clicks the camera select button.  The expectation is that this method retuns the list of available cameras.  The user will choose the camera they want to use, and it may or may not be camera 0.
For example, if their current selection is camera 1, then clicking the camera selection button must not have the side-effect of automatically selecting camera 0.